src/platforms/web/runtime/modules/transition.js

```diff
@@ -280,7 +280,7 @@ export function leave (vnode: VNodeWithData, rm: Function) {
 }
 
 // only used in dev mode
-function checkDuration (val, name, vnode) {
+export function checkDuration (val: number, name: string, vnode: VNode) {
   if (typeof val !== 'number') {
     warn(
       `<transition> explicit ${name} duration is not a valid number - ` +
```

Comment:
This warning should be updated because now it should work on both `<transition>`s and `<transition-group>`s.